From: "Matteo F. Vescovi" <mfv.debian@gmail.com>
Date: Fri, 15 Nov 2013 16:54:50 +0100
Subject: Drop_PHONY_on_pc_files

---
 Makefile | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/Makefile b/Makefile
index e6f08a7..22f756e 100644
--- a/Makefile
+++ b/Makefile
@@ -122,10 +122,6 @@ tmp/$(SYSTEM)/default/shared/glew.o: src/glew.c include/GL/glew.h include/GL/wgl
 	@mkdir -p $(dir $@)
 	$(CC) -DGLEW_NO_GLU $(CFLAGS) $(CFLAGS.SO) -o $@ -c $<
 
-# Force re-write of glew.pc, GLEW_DEST can vary
-
-.PHONY: glew.pc
-
 glew.pc: glew.pc.in
 	sed \
 		-e "s|@prefix@|$(GLEW_DEST)|g" \
@@ -162,10 +158,6 @@ tmp/$(SYSTEM)/mx/shared/glew.o: src/glew.c include/GL/glew.h include/GL/wglew.h
 	@mkdir -p $(dir $@)
 	$(CC) -DGLEW_NO_GLU -DGLEW_MX $(CFLAGS) $(CFLAGS.SO) -o $@ -c $<
 
-# Force re-write of glewmx.pc, GLEW_DEST can vary
-
-.PHONY: glewmx.pc
-
 glewmx.pc: glew.pc.in
 	sed \
 		-e "s|@prefix@|$(GLEW_DEST)|g" \
